Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -747,46 +750,49 @@
 
             // add a role column for each of the roles
             // note iterating in reverse so we don't have to update the insert index
-            this._.forEachRight(this.itContractRoles, role => {
-                var roleColumn: IKendoGridColumn<IItContractOverview> = {
-                    field: `role${role.Id}`,
-                    title: role.Name,
-                    persistId: `role${role.Id}`,
-                    template: dataItem => {
-                        var roles = "";
+            if (this.uiState.isBluePrintNodeAvailable(uiBluePrint.children.contractRoles)) {
```

Comment:
I'm not sure if it's a good way to "hide" columns which were unselected, but I couldn't find a better way

Reply:
Also is there a column connected to "Advice"?